contents/docs/getting-started/group-analytics.mdx

```diff
@@ -8,25 +8,28 @@ import CreateGroupType from "./\_snippets/group-type-create.mdx"
 import ChatGroupType from "./\_snippets/group-type-create-chat.mdx"
 import SetGroupProperties from "./\_snippets/set-group-properties.mdx"
 
-> **Note: ** This is a paid feature and is not available on the open-source or free cloud plan. Learn more [here](/pricing).
+> **Note: ** This is a paid feature and is not available on the open source or free cloud plan. See our [pricing page](/pricing) for more.
 
-Group analytics is a powerful feature in PostHog that allows you to perform analytics on entities other than single users.
-This feature is slightly more advanced, but opens up entirely new avenues for analytics which are essential for most B2B use cases.
+Group analytics is a powerful feature in PostHog that allows you to perform analytics on entities (e.g. companies, teams etc.) other than single users.
+This feature is slightly more advanced, but opens up entirely new avenues for analytics that are essential for most B2B use cases.
 
-Group analytics can sometimes feel confusing to new users, and this often times comes from skipping over the basic concepts too quickly.
+Group analytics can sometimes feel confusing to new users, and this often comes from skipping over the basic concepts too quickly.
 We'll start with an overview of how Group analytics works, and then jump into creating new groups and viewing them within PostHog.
 
 ## 1. What is a group?
 
 In PostHog, a 'Group' is an entity that we can send events and hold properties that is beyond a single user.
 
-To get a more concrete idea of what this looks like in practice, here are a couple examples of common entities that we can model using groups:
+Here are two examples of common entities we can model using groups:
 
-1. **Organizations/Teams**: It's very common in B2B apps to have multiple users under the same 'Team' account, in which case groups allow us to perform analytics on individual teams exactly as if they were users.
-2. **Threads/Chats/Posts**: If you're creating a social app, you might want to track events that are connected with specific conversations such as replies
+1. **Organizations/Teams**: It's very common in B2B apps to have multiple users under the same 'Team' account, in which case groups allow us to perform analytics on individual teams as if they were users.
```

Comment:
Im not clear what we mean by " as if they were users.". Also, is "perform" the correct verb for analytics? Maybe Investigate?

```suggestion
1. **Organizations/Teams**: It's very common in B2B apps to have multiple users under the same 'Team' account, in which case groups allow us to investigate analytics on individual teams.
```